### DISCUSSION TITLE
[Feature] URL to specific image

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [immich-app/immich#8194] [Feature Request]
### The feature

When you click on a specific asset in the timeline (or anywhere) and it opens in the asset viewer there should be a URL directly to that asset, e.g. `https://[IMMICH_URL]/photos/[ID_HERE]` or something like that. This already exists for albums and is great.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #7017
